.travis.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at:
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+sudo: required
+language: go
+os:
+  - linux
+  - osx
+go_import_path: github.com/bazelbuild/sandboxfs
+go:
+  - 1.8
+  - 1.9
+env:
+  - SANDBOXFS=sandboxfs
+
+before_install:
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then ./.travis/linux_install.sh; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then ./.travis/osx_install.sh; fi
```

Comment:
I'd like the scripts to be different because it makes no sense to have the test for the environment variable (`TRAVIS_OS_NAME`) inside the script, since that is Travis specific.
Using `./.travis/${TRAVIS_OS_NAME}_install.sh` instead of the if condition is fine though.

Reply:
Saw this after committing the internal change... but: these scripts are Travis-specific so it's OK for them to inspect the environment variables that Travis promises. (After all, they are assuming that Linux == Debian and that Homebrew is available on Mac, both of which are false in the general case.)